### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Sandoryk**
## React Boilerplate

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md)
- [ ] double-check your branch is based on `dev` and targets `dev` 
- [ ] Pull request has tests (we are going for 100% coverage!)
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Internal code generators and templates are updated (if necessary)
- [ ] Description explains the issue/use-case resolved and auto-closes related issues

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [MIT License](https://github.com/react-boilerplate/react-boilerplate/blob/master/LICENSE.md).

### CODE REVIEW

File: app/components/Button/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * Button.js
+ *
+ * A common button, if you pass it a prop "route" it'll render a link to a react-router route
+ * otherwise it'll render a link with an onclick
+ */
+
+import React, { Children } from 'react';
+import PropTypes from 'prop-types';
+
+function Button(props) {
```
Comment (Hliebov) [inline app/components/Button/index.js]: use const Button = () => {}  syntax

File: app/components/Caption/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Caption(props) {
```
Comment (Hliebov) [inline app/components/Caption/index.js]: same to all components

File: app/containers/App/index.js [hunk 3]
```diff
@@ -1,29 +1,46 @@
...
 import { Switch, Route } from 'react-router-dom';
 
-import HomePage from 'containers/HomePage/Loadable';
-import NotFoundPage from 'containers/NotFoundPage/Loadable';
+import Footer from 'components/Footer';
+import Caption from 'components/Caption';
+import SearchPanel from 'containers/SearchPanel';
+
+const AppWrapper = styled.div`
+  max-width: calc(768px + 16px * 2);
```
Comment (Hliebov) [inline app/containers/App/index.js]: move to css

File: app/components/Button/index.js (hunk 1 above)
Comment (Sandoryk) [reply app/components/Button/index.js]: Done.

File: app/components/Caption/index.js (hunk 2 above)
Comment (Sandoryk) [reply app/components/Caption/index.js]: Done

File: app/containers/App/index.js (hunk 3 above)
Comment (Sandoryk) [reply app/containers/App/index.js]: Done

**Pull request merged**